pytorch_blade/tests/disc/ops/test_unary_ops.py

```diff
@@ -71,23 +71,23 @@ def tanh_func(x):
 
         self._test_unary_ops(tanh_func)
 
-    @skipIfNoTorchMlir()
+    @skipIfEnableTorchMlir()
     def test_contiguous(self):
         @torch.jit.script
         def contiguous_func(x):
             return x.contiguous()
 
         self._test_unary_ops(contiguous_func)
 
-    @skipIfNoTorchMlir()
+    @skipIfEnableTorchMlir()
```

Comment:
export TORCH_DISC_USE_TORCH_MLIR=true needs to be set to enable the TorchMLIR UTs

Reply:
The `unittest` would run all functions with the prefix `test`, actually, we should SKIP some of them if not implemented in torch-mlir converter.